### PR TITLE
Remove the cli command ``neurom features``

### PR DESCRIPTION
**Pull request by asanin-epfl**
Instead a proper documentation is provided on the list of available features.

### CODE REVIEW

File: neurom/features/neuronfunc.py [hunk 1]
```diff
@@ -209,24 +221,25 @@ def _sort_angle(p1, p2):
             for i, _ in enumerate(ordered_vectors)]
 
 
-def sholl_crossings(neurites, center, radii, neurite_filter=None):
+def _sholl_crossings(neurites, center, radii, neurite_type=NeuriteType.all):
```
Comment (asanin-epfl) [inline neurom/features/neuronfunc.py:224]: @arnaudon I've made this function private. Is it ok for you? Did you use it?

Comment (arnaudon) [reply neurom/features/neuronfunc.py:224]: ah could you leave it public, we could use it with specific argument for neurite type if recall, thanks!

File: neurom/features/neuronfunc.py [hunk 2]
```diff
@@ -209,24 +221,25 @@ def _sort_angle(p1, p2):
...
         radii(iterable of floats): radii for which crossings will be counted
+        neurite_type(NeuriteType): Type of neurite to use. By default ``NeuriteType.all`` is used.
 
     Returns:
         Array of same length as radii, with a count of the number of crossings
         for the respective radius
+
+    This function can also be used with a list of sections, as follow::
+
+        secs = (sec for sec in nm.iter_sections(neuron) if complex_filter(sec))
```
Comment (arnaudon) [inline neurom/features/neuronfunc.py:239]: @asanin-epfl , this use case in fact.

**Review by arnaudon — CHANGES_REQUESTED** [review]
Just put back sholl to pubic, then all good!

**Review by arnaudon — APPROVED** [review]
Thank you!

**Comment by asanin-epfl** [comment]
Thank you!

**Pull request merged**